src/data/data.ts

```diff
@@ -322,6 +322,12 @@ export const RiskProfile: { [key: string]: PersonRiskValue } = {
   },
 }
 
+// This is an explicit list of risk profiles for which "their vaccine" modifiers
+// apply.
+export const VaccineModifiableRiskProfiles: (keyof typeof RiskProfile)[] = Object.keys(
```

Comment:
Can we remove this?

Reply:
I can conceive of a future in which we eventually add risk profiles for vaccinated people. Also this line is set in a spot where it picks up everything before the special RiskProfileEnum profiles.

Reply:
> Also this line is set in a spot where it picks up everything before the special RiskProfileEnum profiles.

Oh, yeep, that's super easy to miss.

If the vast majority of profiles are modifiable, I think it would make more sense to have an explicit list of profiles that don't get modified, which would be the keys of RiskProfileEnum below (which is not a very descriptive name).

Reply:
Renamed RiskProfileEnum to RiskProfilesUnaffectedByVaccines.

Reply:
(also it would be the values of RiskProfileEnum, rather than the keys)